botCommands/__snapshots__/points.test.js.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`/leaderboard callback returns correct output 1`] = `
+"**leaderboard** 
+1 - user1 [2508 points] :tada: 
+2 - user2 [1769 points] 
+3 - user3 [1714 points] 
+4 - user4 [1377 points] 
+5 - user5 [1270 points] 
+"
+`;
+
+exports[`@user ++ callback returns correct output for a single user entering club-40 1`] = `"HEYYY EVERYONE SAY HI TO [object Object] the newest member of CLUB 40"`;
+
+exports[`@user ++ callback returns correct output for a single user entering club-40 2`] = `"Woot! [object Object] now has 40 points"`;
+
+exports[`@user ++ callback returns correct output for a single user w/o club-40 1`] = `"Sweet! [object Object] now has 21 points"`;
+
+exports[`@user ++ callback returns correct output for a user mentioning Odin Bot 1`] = `"awwwww shucks... :heart_eyes:"`;
+
+exports[`@user ++ callback returns correct output for a user mentioning themselves 1`] = `"http://media0.giphy.com/media/RddAJiGxTPQFa/200.gif"`;
+
+exports[`@user ++ callback returns correct output for a user mentioning themselves 2`] = `"You can't do that!"`;
+
+exports[`@user ++ callback returns correct output for more than five mentioned users 1`] = `"you can only do 5 at a time..... "`;
+
+exports[`@user ++ callback returns correct output for more than five mentioned users 2`] = `"Sweet! [object Object] now has 11 points"`;
```

Comment:
Should we be receiving `[object Object]` here?

Reply:
So we don’t “want” it, but then I run into an issue if I was to change the mocks to return say, the username or user id. Part of the issue is the string being sent in channel.send is this 

```   
`${exclamation(pointsUser.points)} ${user} now has ${
              pointsUser.points
            }
```

`user` is always going to be an object (in this case, Discord’s special class is handled some way and we just see the mention on the real thing). I have to return an object here because we use the user object properties and methods in other places. I could try to do something similar to what I did with other mentions based callbacks, like I did with /time, but that didn’t mock the class, it actually uses instances of DiscordJS’ actual class. And even then, the /time command handles this in a way where the user ID is simply referenced and mentioned, whereas here it looks like the entire user object is being filtered out with filter or find earlier in the function.